ini/stm32f1.ini

```diff
@@ -364,20 +329,20 @@ upload_protocol = serial
 # Longer 3D board in Alfawise U20 (STM32F103VET6)
 #
 [env:STM32F103VE_longer]
-platform            = ${common_stm32.platform}
-extends             = common_stm32
-board               = genericSTM32F103VE
-board_build.core    = stm32
-board_build.variant = MARLIN_F103Vx
-board_build.offset  = 0x10000
-board_build.address = 0x08010000
-build_flags         = ${common_stm32.build_flags} -DMCU_STM32F103VE -DU20 -DTS_V12
-  -DLED_BUILTIN=PC2 -UPIN_WIRE_SDA -UPIN_WIRE_SCL -DPIN_WIRE_SDA=PB11 -DPIN_WIRE_SCL=PB10
-  -DHAL_DAC_MODULE_DISABLED -DHAL_I2S_MODULE_DISABLED
-build_unflags       = ${common_stm32.build_unflags}
-  -DUSBCON -DUSBD_USE_CDC -DHAL_PCD_MODULE_ENABLED
-extra_scripts       = ${stm32f1_variant.extra_scripts}
-  buildroot/share/PlatformIO/scripts/STM32F103VE_longer.py
+platform                    = ${common_stm32.platform}
+extends                     = stm32_variant
+board                       = genericSTM32F103VE
+board_build.variant         = MARLIN_F103Zx
```

Comment:
sigh, stop breaking my board :p its a Vx

Reply:
Indeed! And I would also add that devs need to make sure to merge the latest code before starting new work, otherwise they will introduce regressions.

Reply:
Whoops, this may have been from my rebase. Apologies!

Reply:
no problem, still have work to do to make it boot ;)